### ISSUE TITLE
SNOW-173611: SNOW-101830: How to use?

### ISSUE THREAD

**Issue by PierrickI3** [snowflakedb/snowflake-connector-nodejs#113]
What is the proper way to use the changes submitted by SNOW-101830?

Is it enough to pass the tokens to createConnection() like the following?

```
const connection = snowflake.createConnection({
  sessionToken: 'SESSION_TOKEN',
  sessionTokenExpirationTime: 1594206508568,
  masterToken: 'MASTER_TOKEN',
  masterTokenExpirationTime: 1594217308568,
});
```

When I run this, I get the following error: `Failure: An accessUrl must be specified.`

Since I use this connector from lambda to read data from our databases, I would like to be able to re-use a connection without having to issue new tokens for every request.

**Comment by PierrickI3** [comment]
@sfc-gh-abhatnagar Could you help?

**Comment by sfc-gh-abhatnagar** [comment]
Hi @PierrickI3 , I will suggest you to connect with Snowflake Support on this . they are better equipped to guide and help you  for such requests.

**Comment by sfc-gh-hkapre** [comment]
@PierrickI3 We are planning to add support for connection pooling in the October timeframe. I think this should address your use case. Let me know if that is correct.

**Comment by PierrickI3** [comment]
Thanks. It will help!

**Comment by sfc-gh-dszmolka** [comment]
connection pooling capability released with [v1.6.3](https://github.com/snowflakedb/snowflake-connector-nodejs/releases/tag/v1.6.3), closing this issue
